pkg/network/proxy.go

```diff
@@ -51,3 +82,65 @@ func validateKubeProxy(conf *operv1.NetworkSpec) []error {
 
 	return out
 }
+
+// FillKubeProxyDefaults inserts kube-proxy defaults, but only if
+// kube-proxy will be deployed explicitly.
+func FillKubeProxyDefaults(conf, previous *operv1.NetworkSpec) {
+	if conf.DeployKubeProxy == nil {
+		v := ShouldDeployKubeProxy(conf)
+		conf.DeployKubeProxy = &v
+	}
+
+	if !*conf.DeployKubeProxy {
+		return
+	}
+
+	if conf.KubeProxyConfig == nil {
+		conf.KubeProxyConfig = &operv1.ProxyConfig{}
+	}
+
+	if conf.KubeProxyConfig.BindAddress == "" {
+		conf.KubeProxyConfig.BindAddress = "0.0.0.0"
+	}
+}
+
+// IsKubeProxyChangeSafe is noop, but it would check if the proposed kube-proxy
+// change is safe.
+func IsKubeProxyChangeSafe(prev, next *operv1.NetworkSpec) []error {
+	// At present, all kube-proxy changes are safe to deploy
+	return nil
+}
+
+// RenderStandaloneKubeProxy renders the standalone kube-proxy if installation was
+// requested.
+func RenderStandaloneKubeProxy(conf *operv1.NetworkSpec, manifestDir string) ([]*uns.Unstructured, error) {
+	if !*conf.DeployKubeProxy {
+		return nil, nil
+	}
+
+	kpcDefaults := map[string][]string{
+		"metrics-bind-address": {"0.0.0.0"},
+		"metrics-port":         {"9101"},
+		"proxy-mode":           {"iptables"},
+	}
+
+	kpc, err := kubeProxyConfiguration(conf, kpcDefaults)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to generate kube-proxy configuration file")
+	}
+
+	// render the manifests on disk
```

Comment:
this comment keeps getting copied into various files and is wrong everywhere

Reply:
it's not wrong, it's just... misleading. which makes it wrong.